pom.xml

```diff
@@ -30,13 +30,25 @@
     </developers>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
+        <jenkins.version>1.596.1</jenkins.version>
```

Comment:
JFYI:
I suppose you migrated to Jenkins 1.590 for `Jenkins#getActiveInstance`, which suppresses errors of findbugs for `CheckForNull` annotating`Jenkins#getInstance`.
`Jenkins#getActiveInstance` is deprecated in Jenkins-1.653 and `Jenkins#getInstance` is annotated with `CheckForNull` now. `Jenkins#getInstanceOrNull` is introduced instead.